### PR TITLE
Add typing spring

### PR DESCRIPTION
**Pull request by CarliJoy**
I never have created a PEP nor attended a sprint but really would like to bring these topics forward.

### CODE REVIEW

**Comment by rodrigogiraoserrao** [comment]
Hey there @CarliJoy, thanks for your enthusiasm and willingness to organise a sprint around this topic.

I like your proposal but this initiative may be more appropriate as an open space during the conference.
I'll check with the team and I'll get back to you ASAP.

File: data/pages-content/sprints.md [hunk 1]
```diff
@@ -184,6 +184,18 @@ Please add your sprint here!
...
+
+- Number of people: ~5 (1 organiser)
+- Room: RB 113
+- Python Level: experience in typing recommended
+- Contact: Carli Freudenberg ([Email](mailto:kound+ep2023@posteo.de), [GitHub](https://github.com/CarliJoy))
+- Links:
+   - [Intersection Operator](https://github.com/python/typing/issues/213)
+   - [Not Operator](https://github.com/python/typing/issues/801)
+   - [copy_kwargs function](https://stackoverflow.com/questions/71968447/python-typing-copy-kwargs-from-one-function-to-another)
+
```
Comment (rodrigogiraoserrao) [inline data/pages-content/sprints.md:198]: Please move this to the _end_ of the sprint list :)

**Review by rodrigogiraoserrao — CHANGES_REQUESTED** [review]

**Comment by rodrigogiraoserrao** [comment]
@CarliJoy turns out everyone thinks your proposal is a good idea 🙌 

I requested a change in your PR before I can approve this and merge.
I'll be on standby.

File: data/pages-content/sprints.md (hunk 1 above)
Comment (CarliJoy) [reply data/pages-content/sprints.md]: Changed :)

**Comment by CarliJoy** [comment]
@rodrigogiraoserrao Sounds great. Looking forward to it.
Sorry for my late reply. Somehow didn't recognize this before.

**Comment by rodrigogiraoserrao** [comment]
> @rodrigogiraoserrao Sounds great. Looking forward to it.
> Sorry for my late reply. Somehow didn't recognize this before.

No worries :)

**Review by rodrigogiraoserrao — APPROVED** [review]

**Comment by CarliJoy** [comment]
Thanks for merging it @rodrigogiraoserrao.
It says that the deployment has succeeded on main but I don't see it on https://ep2023.europython.eu/sprints yet.
I would like to share the link to the sprint, when does it go online?

**Comment by CarliJoy** [comment]
Ah okay, I see on `main` Vercel – ep2023 — Deployment has failed 🤷🏻

**Pull request merged**